AsyncDisplayKit/ASCellNode+Internal.h

```diff
@@ -29,6 +29,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)nodeDidRelayout:(ASCellNode *)node sizeChanged:(BOOL)sizeChanged;
 
+/// Returns the constrained size for the given node
+- (ASSizeRange)constrainedSizeForNode:(ASCellNode *)node;
```

Comment:
We should remove both of these methods.  This one isn't currently used so we should add it once it is needed - if it were called for every cell it would be less efficient than today, so even once we have the formalized node container API we should try to avoid having it "pull" the constrainedSize rather than "push" like node parents do to children.